gef.py

```diff
@@ -8310,13 +8310,23 @@ def do_invoke(self, _: List[str], **kwargs: Any) -> None:
             return
 
         if gef.config["context.grow_stack_down"] is True:
-            from_insnum = nb * (self.repeat_count + 1) - 1
-            to_insnum = self.repeat_count * nb - 1
-            insnum_step = -1
+            if nb > 0:
+                from_insnum = nb * (self.repeat_count + 1) - 1
+                to_insnum = self.repeat_count * nb - 1
+                insnum_step = -1
+            else:
+                from_insnum = 0 + self.repeat_count * nb
```

Comment:
Why `0 +`?

Reply:
The previous implementation had them, so I simply left them in place. I can't tell why they were placed, so I just left them there in case there's some obsure edge-case that I'm not aware of.